doc/smart-switch/high-availability/smart-switch-ha-dpu-scope-dpu-driven-setup.md

```diff
@@ -159,6 +181,18 @@ Here are how the workflows look like for the typical planned operations:
 
 ### 8.1. HA set creation
 
+The HA bring-up workflow is described below -
+  - The DPU starts out with it's initial HA scope Role as Dead.
+  - First the SDN controller pushes all configurations including the HA set and the HA scope with role set to Active but AdminState set to Disabled.
```

Comment:
this is confusing as we are mixing active-standby and active-active.

Reply:
No this should be common irrespective of A-A or A-S. Updated text to specify role could be Active/Standby.

Reply:
not really, because from the description, we are going to set the standby side also as active role. this doesn't look right.

Reply:
looks already updated now. resolving the comment.